src/sentry/static/sentry/app/utils/advancedAnalytics.tsx

```diff
@@ -0,0 +1,107 @@
+import * as qs from 'query-string';
+
+import {Organization} from 'app/types';
+import {trackAnalyticsEvent} from 'app/utils/analytics';
+import {uniqueId} from 'app/utils/guid';
+import {
+  eventNameMap as integrationEventMap,
+  EventParameters as IntegrationEventParameters,
+} from 'app/utils/integrationEvents';
```

Comment:
Hmm I thought you moved these into their own file?

Reply:
Oh I see

Reply:
@EvanPurkhiser I did, `integrationEvents`. That's the file hahah